cv_next/app/not-found.tsx

```diff
@@ -0,0 +1,16 @@
+
+import notFound404 from "../public/images/404.png"
+import notFound404Light from "../public/images/404Light.png"
+import Image from "next/image"
+
+export default function NotFound() {
+    // const { theme, setTheme } = useTheme() 
```

Comment:
Don't leave dangling comments.

BTW, can't the theme control the choice between `NotFound404` and `NotFound404Light`?

Reply:
need to see how to use it, i want to improve all of the theme staff later but for now i think using the tailwind is good